File: app/src/main/java/com/moh/recipes/Recipe.java
package com.moh.recipes;

@Entity(tableName = "recipes")
public class Recipe {
    @PrimaryKey(autoGenerate = true)
    private int rid;

    private String recipe_name;
    private String recipe;

    public Recipe(String recipe_name, String recipe) {
        this.recipe_name = recipe_name;
        this.recipe = recipe;
    }

    public int getRid() {
        return rid;
    }

    public void setRid(int rid) {
        this.rid = rid;
    }

    public String getRecipe_name() {
        return recipe_name;
    }

    public void setRecipe_name(String recipe_name) {
        this.recipe_name = recipe_name;
    }

    public String getRecipe() {
        return recipe;
    }

    public void setRecipe(String recipe) {
        this.recipe = recipe;
    }
}

File: app/src/main/java/com/moh/recipes/Recipe.kt
package com.moh.recipes

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "recipes")
data class Recipe (
    var recipe_name: String,
    var recipe: String
) {
    @PrimaryKey(autoGenerate = true)
    var rid: Int = 0
}